src/utils/__tests__/legacyLayerParamUtils.spec.js

```diff
@@ -200,63 +244,21 @@ describe('Test parsing of legacy URL param into new params', () => {
             it('does not parse an external layer if it is in the current format', () => {
                 const wmtsResult = getLayersFromLegacyUrlParams(
                     fakeLayerConfig,
-                    'WMTS|https://url.to.wmts.server|layer.id|LayerName'
+                    'WMTS|https://url.to.wmts.server|layer.id|LayerName',
+                    undefined,
+                    undefined,
+                    undefined
                 )
                 expect(wmtsResult).to.be.an('Array').empty
                 const wmsResult = getLayersFromLegacyUrlParams(
                     fakeLayerConfig,
-                    `WMS|${'https://wms.server.url?PARAM1=x&'}|layer.id|5.4.3|LayerName`
+                    `WMS|${'https://wms.server.url?PARAM1=x&'}|layer.id|5.4.3|LayerName`,
+                    undefined,
+                    undefined,
+                    undefined
                 )
                 expect(wmsResult).to.be.an('Array').empty
             })
         })
     })
-    describe('test isLayersUrlParamLegacy', () => {
```

Comment:
Here it would be good to test the new `isLegacyParams()` and `parseOpacity` methods.